### ISSUE TITLE
How to load new url correctly

### ISSUE THREAD

**Issue by vanminhquangtri** [google/shaka-player#10167]
### FAQ

- [x] I have read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues

### If the question is related to FairPlay, have you read the tutorial?

Not applicable

### What version of Shaka Player are you using?

4.15.4

### What browser and OS are you using?

Chrome, Edge, Safari

### Please ask your question

The problem only occurs with DRM content.
Step 1: Load the first manifest DRM, play okay.
Step 2: When player throw an error, such as 1002, I load a new second manifest, also DRM, same config with 1st manifest.

Currently: player throw error 3014, all DRM request, manifest request, segment requests are all success (status 200).
I already detach and attach video element again but this not help.

```
 await window.shakaPlayer.detach();  
 await window.shakaPlayer.attach(video);
 await window.shakaPlayer.load(newManifest)
```

The only way I can do know is totally un-mount the component player, which call player.destroy(). But I don't refer this because this make everything must start from begining again. What I want is still keep player instance, only change the manifest.
Can you please give me an advice how to switch manifest correctly.

**Comment by pumbas600** [comment]
@vanminhquangtri I have found that calling `await shakaPlayer.unload()` before loading the new manifest is necessary to prevent the old manifest interferring with the new manifest load. I wasn't getting 3014 errors so this might not help, but it's worth a try. So it would look like:

```ts
// It's not necessary to detect/reattach the video.
await window.shakaPlayer.unload()
await window.shakaPlayer.load(newManigest)
```

**Comment by vanminhquangtri** [comment]
Thanks @pumbas600
